### PR TITLE
Removed LWRP

### PR DESCRIPTION
**Pull request by phi-lira**
### Purpose of this PR
 Removed LWRP dependency from URP package. This means that users can't upgrade from LWRP to URP 2021.2.
 They must first upgrade from LWRP to 11.x.x, then from 11.x.x to 2021.2
 
 This information was added to changelogs and upgrade guide.

---
### Testing status
 Tested locally

---
### Comments to reviewers
 The impact of this PR is that users cannot upgrade from URP to 2021.1 anymore. They will have to go to LWRP -> 11.x.x or earlier -> 2021.2 URP.

### CODE REVIEW

**Review by marctem — APPROVED** [review]
Looks good from Shader Graph side.

File: com.unity.render-pipelines.universal/CHANGELOG.md [hunk 1]
```diff
@@ -13,6 +13,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Fixed an issue where render scale was breaking SSAO in scene view. [case 1296710](https://issuetracker.unity3d.com/issues/ssao-effect-floating-in-the-air-in-scene-view-when-2-objects-with-shadergraph-materials-are-on-top-of-each-other)
 - Fixed GC allocations from XR occlusion mesh when using multipass.
 
+### Removed
+- Code to upgrade from LWRP to URP was removed. This means if you want to upgrade from LWRP you must first upgrade to previous versions of URP and then upgrader to this version.
```
Comment (AliMohebali) [inline com.unity.render-pipelines.universal/CHANGELOG.md:17]: URP and then **upgrade** to this version.

Comment (phi-lira) [reply com.unity.render-pipelines.universal/CHANGELOG.md:17]: good catch.

**Review by theo-pnv — APPROVED** [review]
Yamato wise this looks good.
There are still a few files with lightweight references:
- In the Readme (the CI badges)
- https://github.com/Unity-Technologies/Graphics/blob/universal/remove-lwrp/.github/labeler.yml#L15
- https://github.com/Unity-Technologies/Graphics/blob/universal/remove-lwrp/com.unity.postprocessing/PostProcessing/Runtime/Unity.Postprocessing.Runtime.asmdef#L9 (not sure if this one is important)
- https://github.com/Unity-Technologies/Graphics/blob/universal/remove-lwrp/.github/CODEOWNERS#L22

**Review by martint-unity — COMMENTED** [review]
LGTM

**Review by UnityPaul — APPROVED** [review]

**Review by theo-pnv — APPROVED** [review]

**Review by oleks-k — APPROVED** [review]
Approved with the changes to the upgrade page.

**Review by martint-unity — APPROVED** [review]

**Review by hdb-unity — APPROVED** [review]
Tested by upgrading a (formerly) Lightweight RP project and checking a Universal RP project.

PR looks good. 1x non-PR bug logged.

If a user upgrades a project that still uses LWRP (they did not complete the full LWRP->URP upgrades steps) they may encounter C# or shader errors when upgrading to this version.

The solution is:
- make a project backup before upgrade
- complete the LWRP->URP upgrade process before upgrade

**Pull request merged**